<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetEventItem</class>
 <widget class="QWidget" name="WidgetEventItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>149</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>149</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Event</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#WidgetEventItem {
  background: transparent;
  border: 0;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>9</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBoxMain">
     <property name="styleSheet">
      <string notr="true">#groupBoxMain {
  border-radius: 0px;
  border: 1px solid rgb(131, 147, 167);
  background-color: rgb(80,112,125);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QWidget" name="widgetVotingTime" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#widgetVotingTime {
  color: white;
  background-color: rgb(252, 233, 79);
  background: rgb(252, 233, 79);
  border: 2px solid rgb(181,197, 217);
  border-radius: 5px;
}</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Roboto</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(28,52,62);</string>
             </property>
             <property name="text">
              <string>Voting Time</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,0,0,0,0">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="labelPhotoIcon">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelHead">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;</string>
            </property>
            <property name="text">
             <string>Event Name</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonNotification">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="toolTip">
             <string>Event data was updated, click to referesh!</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
  background-color: transparent;
  border-radius: 10px;
  border-image: url(:/icon-notify.png);
}
QPushButton:hover {
  background-color: rgb(50,82,95);
  border-radius: 10px;
}</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonNewLocation">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="toolTip">
             <string>Create a new location</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
  background-color: transparent;
  border-radius: 10px;
  border-image: url(:/icon-location-small.png);
}
QPushButton:hover {
  background-color: rgb(50,82,95);
  border-radius: 10px;
}</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonDelete">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="toolTip">
             <string>Delete Event</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
  background-color: transparent;
  border-radius: 10px;
  border-image: url(:/icon-delete-small.png);
}
QPushButton:hover {
  background-color: rgb(50,82,95);
  border-radius: 10px;
}</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonEdit">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="toolTip">
             <string>Event settings</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
  background-color: transparent;
  border-radius: 10px;
  border-image: url(:/icon-settings-small.png);
}
QPushButton:hover {
  background-color: rgb(50,82,95);
  border-radius: 10px;
}</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonCollapse">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="toolTip">
             <string>Hide/show details</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
  border-image: url(:/icon-collapse2.png);
}
QPushButton:checked {
  border-image: url(:/icon-collapse1.png);
}</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widgetBody" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>80</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0">
             <property name="spacing">
              <number>2</number>
             </property>
             <item>
              <widget class="QLabel" name="labelLine">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>2</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>2</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QLabel {
  background-color: rgb(131,147, 167);
}</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="topMargin">
                <number>3</number>
               </property>
               <item>
                <widget class="QLabel" name="labelDescription">
                 <property name="font">
                  <font>
                   <family>Roboto</family>
                   <pointsize>9</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: white;</string>
                 </property>
                 <property name="text">
                  <string>Description</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelPhoto">
                 <property name="minimumSize">
                  <size>
                   <width>64</width>
                   <height>64</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>64</width>
                   <height>64</height>
                  </size>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonEdit</sender>
   <signal>clicked()</signal>
   <receiver>WidgetEventItem</receiver>
   <slot>onBtnEditClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonNewLocation</sender>
   <signal>clicked()</signal>
   <receiver>WidgetEventItem</receiver>
   <slot>onBtnNewLocationClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonNotification</sender>
   <signal>clicked()</signal>
   <receiver>WidgetEventItem</receiver>
   <slot>onBtnNotificationClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDelete</sender>
   <signal>clicked()</signal>
   <receiver>WidgetEventItem</receiver>
   <slot>onBtnDeleteClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCollapse</sender>
   <signal>toggled(bool)</signal>
   <receiver>WidgetEventItem</receiver>
   <slot>onBtnCollapseToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBtnEditClicked()</slot>
  <slot>onBtnNewLocationClicked()</slot>
  <slot>onBtnNotificationClicked()</slot>
  <slot>onBtnDeleteClicked()</slot>
  <slot>onBtnCollapseToggled(bool)</slot>
 </slots>
</ui>
